### PR TITLE
Added median of operations progress

### PR DESCRIPTION
**Pull request by d2dyno1**
**Resolved / Related Issues**
\---

**Details of Changes**
Adds median percentage value of all operations.

**Validation**
How did you test these changes?
- [x] Built and ran the app

### CODE REVIEW

**Review by winston-de — APPROVED** [review]

**Comment by d2dyno1** [comment]
wait

**Comment by yaichenbaum** [comment]
@d2dyno1 Please fill out the PR template.

File: Files/ViewModels/StatusCenterViewModel.cs [hunk 1]
```diff
@@ -87,24 +94,37 @@ public bool CloseBanner(StatusBanner banner)
             return true;
         }
 
-        #endregion IStatusCenterActions
-
-        #region INotifyPropertyChanged
```
Comment (winston-de) [inline Files/ViewModels/StatusCenterViewModel.cs:97]:
```suggestion
```
Minor CodeFactor thing.

**Review by winston-de — APPROVED** [review]

File: Files/App.xaml.cs [hunk 2]
```diff
@@ -446,6 +446,42 @@ private void OnSuspending(object sender, SuspendingEventArgs e)
...
+            }
+            else
+            {
+                Debug.WriteLine("\nException is null!\n");
+            }
+
+            Debug.WriteLine("---------------------------------------");
+
+            Debugger.Break(); // Please check "Output" Window for exception details (View -> Output Window) (CTRL + ALT + O)
+#endif
```
Comment (yaichenbaum) [inline Files/App.xaml.cs:483]: Can this be done in a separate pull request?

Comment (d2dyno1) [reply Files/App.xaml.cs:483]: Is it necessary?

Comment (yaichenbaum) [reply Files/App.xaml.cs:483]: Yes, it's something we are going to be enforcing more in order to better track changes.

Comment (d2dyno1) [reply Files/App.xaml.cs]: Ok

**Review by yaichenbaum — APPROVED** [review]
LGTM
